source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.Android/SKSwapChainPanel.Android.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace SkiaSharp.Views.UWP
+{
+	public partial class SKSwapChainPanel : FrameworkElement
+	{
+		private SKGLTextureView glTextureView;
+
+		public SKSwapChainPanel()
+		{
+			Initialize();
+		}
+
+		private SKSize GetCanvasSize() =>
+			glTextureView?.CanvasSize ?? SKSize.Empty;
+
+		private GRContext GetGRContext() =>
+			glTextureView?.GRContext;
+
+		partial void DoLoaded()
+		{
+			glTextureView = new SKGLTextureView(Context);
+			glTextureView.PaintSurface += OnPaintSurface;
+			AddView(glTextureView);
```

Comment:
Not sure if this is "safe" in Uno, but we need to use the separate view type for hardware acceleration.

Reply:
We _can_ use layers for iOS/macOS, but it will be nice to just use views there too.